aws_fuzzy_finder/main.py

```diff
@@ -14,10 +14,13 @@
     ENV_USE_PUBLIC_DNS_OVER_IP,
     ENV_KEY_PATH,
     ENV_SSH_COMMAND_TEMPLATE,
+    ENV_SSM_COMMAND_TEMPLATE,
+    ENV_USE_SSM,
     ENV_SSH_USER,
     ENV_TUNNEL_SSH_USER,
     ENV_TUNNEL_KEY_PATH,
     AWS_REGIONS,
+    AWS_DEFAULT_PROFILE,
```

Comment:
Had to pull in the default profile from the environment variable so that you could change it needed.